examples/material-ui-nextjs-ts/src/app/page.tsx

```diff
@@ -20,11 +20,11 @@ export default function Home() {
         }}
       >
         <Typography variant="h4" component="h1" sx={{ mb: 2 }}>
-          Material UI - Next.js App Router example in TypeScript
+          Material UI - Next.js example in TypeScript
         </Typography>
-        <Link href="/about" color="secondary" component={NextLink}>
+        <Button component={Link} href="/about" color="secondary">
           Go to the about page
-        </Link>
```

Comment:
We could have used Material UI `Link` and passed Next.js Link (the one one which we separated out in a separate file) to `component`. Why change it to Button?